<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtSE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frameTools">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QToolButton" name="toolButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>28</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButton_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>10</y>
          <width>28</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButton_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>10</y>
          <width>28</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButton_4">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>28</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>521</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport_as">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionAnimated_GIF"/>
     <addaction name="actionPNG_Sequence"/>
     <addaction name="actionAll_Frames"/>
     <addaction name="actionSprite_Sheet"/>
    </widget>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="action32x32"/>
     <addaction name="action16x16"/>
     <addaction name="action32x32_2"/>
     <addaction name="action64x64"/>
     <addaction name="action128x128"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="menuExport_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="statusTip">
    <string>Perhaps we can show the name of the tool here.</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAnimated_GIF">
   <property name="text">
    <string>Animated GIF</string>
   </property>
  </action>
  <action name="actionPNG_Sequence">
   <property name="text">
    <string>Current Frame</string>
   </property>
  </action>
  <action name="actionAll_Frames">
   <property name="text">
    <string>All Frames</string>
   </property>
  </action>
  <action name="actionSprite_Sheet">
   <property name="text">
    <string>Sprite Sheet</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionResize_Canvase">
   <property name="text">
    <string>Resize Canvas</string>
   </property>
  </action>
  <action name="action32x32">
   <property name="text">
    <string>8x8</string>
   </property>
  </action>
  <action name="action16x16">
   <property name="text">
    <string>16x16</string>
   </property>
  </action>
  <action name="action32x32_2">
   <property name="text">
    <string>32x32</string>
   </property>
  </action>
  <action name="action64x64">
   <property name="text">
    <string>64x64</string>
   </property>
  </action>
  <action name="action128x128">
   <property name="text">
    <string>128x128</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
